Lib/fontTools/feaLib/ast.py

```diff
@@ -1306,20 +1553,24 @@ def __init__(self, character, tag, location=None):
         self.tag = tag
 
     def build(self, builder):
+        """Calls the builder object's ``add_cv_character`` callback."""
         builder.add_cv_character(self.character, self.tag)
 
     def asFea(self, indent=""):
         return "Character {:#x};".format(self.character)
 
 
 class BaseAxis(Statement):
+    """An axis definition, being either a ``VertAxis.BaseTagList/BaseScriptList``
+    pair or a ``HorizAxis.BaseTagList/BaseScriptList`` pair."""
     def __init__(self, bases, scripts, vertical, location=None):
         Statement.__init__(self, location)
-        self.bases = bases
-        self.scripts = scripts
-        self.vertical = vertical
+        self.bases = bases #: A list of baseline tag names as strings 
+        self.scripts = scripts #: A list of script record tuplets (script tag, default baseline tag, base coordinate)
+        self.vertical = vertical #: Boolean; VertAxis if True, HorizAxis if False
```

Comment:
It needn’t be a Boolean: any truth value will do.

Reply:
that's what `bool`s are for. We don't want to support setting `vertical="yes"`, even though it happens to work